### PR TITLE
Add Funeral Director Pompes Funèbres Générales

### PR DESCRIPTION
**Pull request by bmillemathias**
PFG: https://overpass-turbo.eu/s/HrX
Pompes Funèbres Générales: https://overpass-turbo.eu/s/HrY

### CODE REVIEW

**Comment by Adamant36** [comment]
Hi there. If they are both tagged with the name tag, you should just pick whatever one has the most usage and add then other one as a match instead of using the brand. Either that or go with the shorter name, as it shows up better on a map, and use the official_name tag for the longer one. Which is what id do in this case.

**Comment by matkoniecz** [comment]
>  you should just pick whatever one has the most usage

Rather the better one (almost always correct `name` tag value - one used by people to refer to the object is the most common one but in some situations it is not happening).

**Comment by matkoniecz** [comment]
To confirm:

`name=PFG` should be used in OSM rather than `name=Pompes Funèbres Générales`, right?

**Comment by bmillemathias** [comment]
> To confirm:
> 
> `name=PFG` should be used in OSM rather than `name=Pompes Funèbres Générales`, right?

From my experience, ads mention PFG as the name so I would use PFG as I did in my commit.

**Comment by Adamant36** [comment]
"From my experience, ads on TV mentioned PFG as the name so I would use PFG as I did in my commit."

That's what I thought. In your commit it says the name tag is "Pompes Funèbres Générales". 

So you should update it to have both the name and brand tags be PFG. Then as I said above you can use official_name for Pompes Funèbres Générales or just add it as a match if there's any usage of it in OSM. 

"Rather the better one"

Which is what I said, in this situation PFG. I didn't say disregard everything else for usage numbers. It is a factor though. So the names length and how people tag it. Both of which I also mentioned.

File: brands/shop/funeral_directors.json [hunk 1]
```diff
@@ -1,4 +1,18 @@
...
+    "countryCodes": ["fr"],
+    "match": [
+      "shop/funeral_directors|Pompes Funèbres Générales"
+    ],
+    "nocount": true,
+    "tags": {
+      "brand": "PFG",
+      "brand:wikidata": "Q3396087",
+      "brand:wikipedia": "fr:Pompes funèbres générales",
+      "name": "Pompes Funèbres Générales",
```
Comment (matkoniecz) [inline brands/shop/funeral_directors.json]: It seems that it should be

```
"official_name": "Pompes Funèbres Générales",
"name": "PFG",
```

according to other data and discussion.

Comment (bhousel) [reply brands/shop/funeral_directors.json]: I merged and did this..

**Pull request merged**